src/culc/TestCulc.java

```diff
@@ -0,0 +1,138 @@
+package culc;
+
+import java.util.Scanner;
+
+public class TestCulc {
+
+    public static String[] romanNumber = {"I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX", "X", "XX", "XXX", "XL", "L",
+    "LX","LXX","LXX","XC","C","CC","CCC","CD","D","DC","DCC","DCCC","CM","M"};
+
+
+    public static void main(String[] args) {
+
+        String[] romanNumbersOut = new String[100];
+
+        Scanner inNumbers = new Scanner(System.in);
+
+        System.out.println("Enter arabic numbers: ");
+
+        int arabic = inNumbers.nextInt();
+
+        for (int i = 0; i < 5; i++) {
+
+            if (arabic % 10 == 1) {
+                romanNumbersOut[0] = romanNumber[0];
+                arabic -= 1;
```

Comment:
It’s too hard logic. What if you create array for 1-9 Arabic digits ant then just get from array by index?
For example:
If(arabic > 10 && arabic %10 !=0)
 romanNumbersOut = digits[arabic%10]